src/ripple/app/main/LoadManager.cpp

```diff
@@ -32,40 +33,56 @@
 
 namespace ripple {
 
-class LoadManagerImp
-    : public LoadManager
-    , public beast::Thread
+class LoadManagerImp : public LoadManager
 {
 public:
-    using LockType = std::mutex;
-    using ScopedLockType = std::lock_guard <LockType>;
+    //--------------------------------------------------------------------------
 
-    Application& app_;
-    beast::Journal m_journal;
-    LockType mLock;
+    Application& mApp;
+    beast::Journal mJournal;
```

Comment:
I prefer that the naming convention within the class be consistent.  I went with the preceding `m` since that seemed to be the prevailing convention within the class.  I'd be every bit as happy with a trailing underscore, but I prefer that they be consistent.